cli/dcoscli/data/help/cluster.txt

```diff
@@ -17,7 +17,7 @@ Usage:
 
 Commands:
     attach
-        List only the currently attached cluster.
+        Attaches the named cluster.
```

Comment:
To be consistent with the other descriptions in this section, this should be "Attach the named cluster". Else the others should be "Lists CLI...", "Renames a cluster..." etc.